azext_edge/tests/edge/orchestration/resources/test_dataflow_profiles_unit.py

```diff
@@ -21,22 +29,21 @@ def get_dataflow_profile_endpoint(
     resource_path = f"/instances/{instance_name}/dataflowProfiles"
     if profile_name:
         resource_path += f"/{profile_name}"
-    return get_base_endpoint(resource_group_name=resource_group_name, resource_path=resource_path)
+    return get_base_endpoint(
+        resource_group_name=resource_group_name, resource_path=resource_path
+    )
 
 
-def get_mock_dataflow_profile_record(profile_name: str, instance_name: str, resource_group_name: str) -> dict:
+def get_mock_dataflow_profile_record(
+    profile_name: str, instance_name: str, resource_group_name: str
+) -> dict:
     return get_mock_resource(
         name=profile_name,
         resource_path=f"/instances/{instance_name}/dataflowProfiles/{profile_name}",
         properties={
             "diagnostics": {
                 "logs": {
                     "level": "info",
-                    "opentelemetryExportConfig": {"otlpGrpcEndpoint": "", "intervalSeconds": 30, "level": "error"},
```

Comment:
why are we deleting these? wouldn't it be nice to test with more detailed data?

Reply:
oh is this the part that going to be removed in the profile property?

Reply:
these properties don't exist on dataflow profiles. at least not in the 2025-04-01 swagger